pkg/node/stager.go

```diff
@@ -375,7 +375,7 @@ func (s *SCSIStager) connectNVMEDevice(ctx context.Context,
 	wwn string, data scsiPublishContextData) (gobrick.Device, error) {
 	logFields := common.GetLogFields(ctx)
 	var targets []gobrick.NVMeTCPTargetInfo
-	for _, t := range data.iscsiTargets {
+	for _, t := range data.nvmeTargets {
```

Comment:
Just curious, how did it work earlier?

Reply:
Actually we are not using the nvmeTargets for the connection. We use nvme list to make further findings and connect the device. We just use it to see the if NVME sessions exists and since iscsi sessions were also there, the function did not return error.

Reply:
ok

Reply:
ok